cfme/tests/cloud_infra_common/test_provisioning.py

```diff
@@ -267,6 +267,12 @@ def test_provision_from_template_using_rest(appliance, request, provider, vm_nam
         initialEstimate: None
     """
     if auto:
+        if (
+            provider.key == "vsphere65-nested"
+            and BZ(1662638, forced_streams=["5.9", "5.10", "upstream"]).blocks
+        ):
+            pytest.skip("VM provisioning fails due to inaccessible iSCSI datastore file.")
```

Comment:
Cool! I didn't know about `unblock`. But I cannot use `not provider.on_of(VMwareProvider)`, since it'll skip all the vmware provider, and my problem's only with vsphrer65.